pkg/gardenlet/controller/federatedseed/network-policy/networkpolicies_controller.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package network_policy
+
+import (
+	networkingv1 "k8s.io/api/networking/v1"
+)
+
```

Comment:
Btw, why there is no `Add` handler?

Reply:
Because the "allow-to-seed-apiserver" network policy is created by the namespace controller when a Shoot namespace is created/ updated. This would cause a duplicate reconcile.